src/CHANGES.md

```diff
@@ -1,6 +1,6 @@
 Changelog
 
-## [Unreleased](https://github.com/bids-standard/bids-specification/tree/HEAD)
+## [v1.2.2](https://bids-specification.readthedocs.io/en/v1.2.2/) (2020-02-03)
```

Comment:
Can you update the links for all of the other releases?

Also, can we remove the `REL` changelog entries from the top of each release?

Reply:
changed the RTD links - I am hesitant to link to the previous pre-RTD until https://github.com/bids-standard/bids-specification/issues/392 has been resolved so we do not have broken links in previous versions

this can be released for stable but will appear back on latest because of how the changelog is generated (for each merged PR), a separate issue could address enhancing the changelog generation